test/pytest/test_browser.py

```diff
@@ -46,3 +46,22 @@ def test_browser(html):
     subprocess.run(['test/common/tap-cdp', f'{BUILDDIR}/test-server',
                     sys.executable, '-m', *coverage, 'cockpit.bridge', '--debug',
                     f'./qunit/{html}'], check=True, stderr=subprocess.STDOUT)
+
+
+# run test-timeformat.ts in different time zones: west/UTC/east
+@pytest.mark.parametrize('tz', ['America/Toronto', 'Europe/London', 'UTC', 'Europe/Berlin', 'Australia/Sydney'])
```

Comment:
Cool.

`test-timeformat` is also going to run from the normal `test_browser()` path, but I guess that's good: then we get the version without explicit `TZ` set.  It also means that `pytest -k browser` will still hit it (at least for the one case) but won't get bogged down with having to do the exhausting checking.

Reply:
Right, I deliberately didn't ignore it from `test_browser()`, it's also nice to test in your current time zone.